baselines/dev/setup-defaults.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+if [ ! -d $HOME/.pyenv ]
+then
+    # Install pyenv with the virtualenv plugin
+    curl https://pyenv.run | bash 
+
+    # To add the config to the right file (depends on the shell used)
+    rcfile=$HOME/.$(basename $SHELL)rc
+
+    # Add pyenv to $PATH
+    echo 'export PATH="$HOME/.pyenv/bin:$PATH"' >> $rcfile
+
+    # Init pyenv with the shell
+    echo 'eval "$(pyenv init -)"' >> $rcfile
+    echo 'eval "$(pyenv virtualenv-init -)"' >> $rcfile
+    source $rcfile
+else
+    # If pyenv is already installed, check for a newer version
+    pyenv update
```

Comment:
Oh yes, I will add a check, it is a plugin that also updates all the other plugins, I will use it if it exists otherwise I will manually perform the pulls.